dragonfly/world/world.go

```diff
@@ -82,7 +82,7 @@ type World struct {
 	positionCache       []ChunkPos
```

Comment:
I don't think these changes are supposed to be here. Did you revert a commit on accident?

Reply:
No idea, I don't think I did that though

Reply:
Removed those changes